.travis.yml

```diff
@@ -12,6 +9,9 @@ cache:
   directories:
     - $HOME/.composer/cache
 
+services:
+  - mysql
+
```

Comment:
A lot of changes to this file were needed for the e2e tests. The upside is that there isn't much forked code, like a `docker-compose.yml` file.